.setup/install_submitty/install_site.sh

```diff
@@ -230,6 +216,20 @@ fi
 find ${SUBMITTY_INSTALL_DIR}/site/vendor -type d -exec chmod 551 {} \;
 find ${SUBMITTY_INSTALL_DIR}/site/vendor -type f -exec chmod 440 {} \;
 
+# clear old doctrine proxy classes
+if [ -d "${SUBMITTY_INSTALL_DIR}/site/cache/doctrine-proxy" ]; then
+    rm -rf "${SUBMITTY_INSTALL_DIR}/site/cache/doctrine-proxy"
+fi
+# create doctrine proxy classes directory
+mkdir -p ${SUBMITTY_INSTALL_DIR}/site/cache/doctrine-proxy
```

Comment:
I'd consider leaving this block where it was just so it's in the same block of deleting and recreating cache folders, where we'd benefit from DRY that up by using an array and a loop, vs all the repetition.